model-serving/src/main/scala/com/lightbend/modelserving/model/h2o/H2OModel.scala

```
@@ -84,7 +84,11 @@ object H2OModel {
    */
   def fromPrediction(prediction: BinomialModelPrediction): (String, Double) = {
     val probs = prediction.classProbabilities
-    val probability = if (probs.length == 2) probs(1) else 0.0
-    (prediction.label, probability)
+    try {
```

I'm not sure how an exception can be thrown in this case, but if `BinomialModelPrediction` can throw one here, what does that mean? Should we log an error or something rather than silently return a default value?

---

If prediction is null, then it barfs. I did not trace if fr enough to see how this can happen, but did see it.